de.yaml

```diff
@@ -1,131 +1,131 @@
-COUPON_CLAIM: Claim a coupon
-COUPON_CLAIM_NOTHING: You have no coupons available
+AGREE_TO_TOS_AND_PP: Ich stimme den {tos} und {pp}
+ALL_RIGHTS_RESERVED: Alle Rechte vorbehalten
+AMOUNT: Menge
+ARE_YOU_NEW: Bist du neu?
+AUTHORITY: Autorität
+BACK_TO_LOGIN: Zurück zum
+BUY_NWC: NWC kaufen
+CHARACTERS: Charaktere
+CONNECT: Verbinden
+CONNECTIONS: Verbindungen
+COPYRIGHT: Urheberrechte
+COUPON_ADDED: Der Gutschein {coupon} wurde deinem Gutscheininventar hinzugefügt.
+COUPON_CLAIM: Gutschein anfordern
+COUPON_CLAIM_NOTHING: Es sind keine Gutscheine verfügbar.
 COUPON_INFORMATION:
-COUPON_NOT_VALID: The coupon {coupon} is not valid or has already been used
-COUPON_NWC_ADDED: The amount of {coins} NWC has been added to your account.
-COUPON_INVENTORY: Coupon Inventory
-COUPON_PLACEHOLDER: Enter coupon's code
-COUPON_REDEEM: Redeem a coupon
-CUSTOMIZE: Customize
-DISABLED: Disabled
-DISCONNECT: Disconnect
-DOWNLOAD_LAUNCHER: Download Launcher
+COUPON_NOT_VALID: Der Gutschein {coupon} ist ungültig oder wurde bereits eingelöst.
+COUPON_NWC_ADDED: Die {coins} NWC wurden deinem Konto gutgeschrieben.
+COUPON_INVENTORY: Gutscheininventar
+COUPON_PLACEHOLDER: Gutscheincode eingeben
+COUPON_REDEEM: Gutschein einlösen
+CUSTOMIZE: Anpassen
+DISABLED: Deaktiviert
+DISCONNECT: Trennen
+DOWNLOAD_LAUNCHER: Launcher herunterladen
 DOWNLOAD_LAUNCHER_VERSION: Download NosWings Launcher (v.{version})
-DOWNLOAD_LAUNCHER_x64_VERSION: Download x64 version
-DOWNLOAD_LAUNCHER_x86_VERSION: Download x86 version
-DOWNLOAD_LAUNCHER_INFO_MESSAGE: If you have no idea about architecture, please download the x86 version
+DOWNLOAD_LAUNCHER_x64_VERSION: x64 Version herunterladen
+DOWNLOAD_LAUNCHER_x86_VERSION: x86 Version herunterladen
+DOWNLOAD_LAUNCHER_INFO_MESSAGE: Solltest du dich mit Architekturen nicht auskennen, wähle die x86 Version aus
 EMAIL:
-  CONFIRM_NEW: Confirm new email
-  CURRENT: Current email
-  NEW: New email
+  CONFIRM_NEW: Neue Email bestätigen
+  CURRENT: Aktuelle Email
+  NEW: Neue email
   TITLE: Email
-  UPDATE: Update Email
-ENABLED: Enabled
-GAME_ACCOUNTS: Game Accounts
-GENERAL: General
-HOME: Home
-LANGUAGE_CZ: Czech
-LANGUAGE_DE: German
-LANGUAGE_EN: English
-LANGUAGE_ES: Spanish
-LANGUAGE_FR: French
-LANGUAGE_IT: Italian
-LANGUAGE_PL: Polish
-LANGUAGE_TR: Turkish
-LOGIN: Login
-LOGOUT: Logout
-LOGS: Logs
-LOGS_DONATION: Donation Logs
-LOGS_SHOP: Shop Logs
-NO_RECORDS_FOUND: No records found
-NUMBER_OF_CHARACTERS: Number of characters
-PAGINATION_TOTAL_COUNT: Showing from {min} to {max} of {total} entries
+  UPDATE: Email aktualisieren
+ENABLED: Aktiviert
+GAME_ACCOUNTS: Spiel Accounts
+GENERAL: Allgemeines
+HOME: Startseite
+LANGUAGE_CZ: Tschechisch
+LANGUAGE_DE: Deutsch
+LANGUAGE_EN: Englisch
+LANGUAGE_ES: Spanisch
+LANGUAGE_FR: Französisch
+LANGUAGE_IT: Italienisch
+LANGUAGE_PL: Polnisch
+LANGUAGE_TR: Türkisch
+LOGIN: Einloggen
+LOGOUT: Ausloggen
+LOGS: Protokolle
+LOGS_DONATION: Spenden Protokoll
```

Comment:
```suggestion
LOGS_DONATION: Spenden Protokolle
```

Reply:
Handelt es sich hierbei um ein Protokoll, dass alle Spenden in einer Liste umfasst, oder mehrere?
Bei mehreren trifft deins zu, ansonsten würde ich bei Protokoll bleiben.